frontend/src/lib/Quiver.ts

```diff
@@ -37,9 +37,14 @@ interface TableCell {
   content: string
 }
 
-export interface Index {
+export interface Data {
   data: any[][]
-  type: IndexType
+  type: string[]
```

Comment:
Can `type` literally be any string, or can this be expressed with a type union? `type: ("foo" | "bar" | "baz")[]`?

Reply:
Done